### PR TITLE
Add claw bar

### PR DESCRIPTION
**Pull request by ampersand55**
#### Summary

```SUMMARY: Content "Adds claw bar, a small prying tool"```

#### Purpose of change

A smallish tool with 2 prying quality would allow you to pry open doors without carrying a big full size crowbar or ice axe.

#### Describe the solution

Added a claw bar, based on the 10" claw bars DeWalt DWHT55524 and Stanley 55-114, with the weight being the average of the two.

Volume of 1.5c (0.375 l), a bit smaller than a hammer, considering it's flat and mostly 2 dimensional.

Added it to item groups being half as common as crowbar.

Links to products:

https://www.dewalt.com/products/hand-tools/hammers-and-pry-bars/10-claw-bar/dwht55524
http://www.protoindustrial.com/en/support/Documents/108/Proto%20108%20-%20Stanley%20Demolition%20Tools%20(p.980-985).pdf

#### Describe alternatives you've considered

I compared many versions of small prying tools, claw bars, flat bars and molding bars of different sizes, and I concluded that a 10" claw bar would be the smallest tool that could open a door as well as a bent pipe or an ice axe.

Some info about the differences between the different pry bars:

https://www.finehomebuilding.com/2015/05/13/whats-the-difference-demolition-tools

#### Additional context

Screenshot: https://i.imgur.com/XdEaKiK.png

### CODE REVIEW

File: data/json/items/tools.json [hunk 1]
```diff
@@ -2872,6 +2872,24 @@
...
+    "type": "TOOL",
+    "name": "claw bar",
+    "description": "This is a small prying tool with a clawed bend at one end for pulling spikes.  Use it to open locked doors without destroying them or to lift manhole covers.  You could also wield it to bash some heads in.",
+    "weight": 340,
+    "volume": "375 ml",
+    "price": 1000,
+    "to_hit": 1,
+    "bashing": 10,
+    "cutting": 1,
+    "material": [ "steel"],
```
Comment (anothersimulacrum) [inline data/json/items/tools.json]:
```suggestion
    "material": [ "steel" ],
```

**Comment by Petethegoat** [comment]
Are these things pretty abundant? Half as common as crowbars feels too common, though of course I don't have any data to base that on.

**Comment by ampersand55** [comment]
> 
> 
> Are these things pretty abundant? Half as common as crowbars feels too common, though of course I don't have any data to base that on.

You're right. They are not half as common as crowbars, except maybe in the "tools_entry" group used by burglars. Perhaps something like 1/5 as common would be more accurate.

**Comment by ZhilkinSerg** [comment]
In the future please use [web-linter](http://dev.narc.ro/cataclysm/format.html) to lint json files.

**Pull request merged**